gson/src/test/java/com/google/gson/internal/UnsafeAllocatorInstantiationTest.java

```diff
@@ -16,6 +16,7 @@
 package com.google.gson.internal;
 
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
```

Comment:
Import seems to be unused now

Reply:
Now it is used then I mark it as resolved